<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartPythonClass</class>
 <widget class="QWidget" name="SmartPythonClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartPython</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>脚本</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="CodeEditor" name="scriptPlainTextEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="saveScriptPushButton">
           <property name="text">
            <string>保存脚本</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="compileScriptPushButton">
           <property name="text">
            <string>编译脚本</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="runScriptPushButton">
           <property name="text">
            <string>执行脚本</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopScriptPushButton">
           <property name="text">
            <string>停止执行</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>日志</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="logPlainTextEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="smartpython.qrc"/>
 </resources>
 <connections/>
</ui>
